src/components/App/App.jsx

```diff
@@ -7,6 +8,13 @@ const App = () => (
       <h1 className="app__heading">CYF Hotel</h1>
     </header>
     <Bookings />
+    <Footer
+      list={[
```

Comment:
Instead of adding the array of data here, try creating a separate .json file, and then import the data from there into your Footer.jsx